### DISCUSSION TITLE
How do i use custom font files (Without Google Fonts)

### DISCUSSION THREAD

**Discussion by GalaxyGaming2000** [tinymce/tinymce#9161] [General]
i was looking to add a font to the editor
i have it in a folder on the website fonts/customfont.tff
no matter how much i try to get it working its always the default
font.

please help me i have been stuck for hours

**Comment by TheSpyder** [comment]
You need to add the font to the content stylesheet
https://www.tiny.cloud/docs/tinymce/6/add-css-options/#add-css-and-styles-to-the-editor
